as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/PomServerConstants.java

```diff
@@ -13,19 +13,20 @@
 public class PomServerConstants {
 	public static final String JBOSS_190_HOME="jbosstools.test.jboss.home.19.0";
 	public static final String JBOSS_200_HOME="jbosstools.test.jboss.home.20.0";
+	public static final String JBOSS_210_HOME="jbosstools.test.jboss.home.21.0";
 	public static final String JBOSS_EAP_70_HOME="jbosstools.test.jboss.home.eap.7.0";
 	public static final String JBOSS_EAP_71_HOME="jbosstools.test.jboss.home.eap.7.1";
 	public static final String JBOSS_EAP_72_HOME="jbosstools.test.jboss.home.eap.7.2";
 	public static final String JBOSS_EAP_73_HOME="jbosstools.test.jboss.home.eap.7.3";
 	// NEW_SERVER_ADAPTER
 
 	public static final String[] PUBLIC = new String[]{
-			JBOSS_190_HOME, JBOSS_200_HOME
+			JBOSS_190_HOME, JBOSS_200_HOME, JBOSS_210_HOME
```

Comment:
That's the fist one to check with QE because I feel only the last 2 WF versions are listed here